controllers/clusters/cassandra_controller.go

```diff
@@ -721,7 +718,9 @@ func (r *CassandraReconciler) handleUsersDelete(
 		return err
 	}
 
-	l.Info("User has been added to the queue for deletion", "username", u.Name)
+	l.Info("User has been added to the queue for deletion",
+		"User resource", u.Namespace+"/"+u.Name,
```

Comment:
I think Spriptf will be better here

Reply:
fmt.Sprint is good for concatenation of different types. 
I prefer my current implementation as it is simpler and faster.